test/integration/table/table.int.js

```diff
@@ -1,10 +1,28 @@
-describe('Simple visualization', () => {
+const path = require('path');
+const serve = require('@nebula.js/cli-serve'); // eslint-disable-line
+const puppeteerUtil = require('../../utils/puppeteer-util');
+
+describe('Table visualization', () => {
   const content = '.simple-table';
+  let s;
+
+  before(async () => {
+    s = await serve({
+      entry: path.resolve(__dirname, 'sn-table'),
+      open: false,
+      fixturePath: 'test/integration/table',
+    });
+    puppeteerUtil.addListeners(page);
+  });
+
+  after(() => {
+    s.close();
```

Comment:
Should you invoke `puppeteerUtil.removeListeners` from here maybe?

Reply:
Good point. I have adjusted the code to remove the listeners after the test cases are completed.